info/about.html

```diff
@@ -0,0 +1,41 @@
+<html>
+	<head>
+		<title>About Me</title>
+		<link rel="stylesheet" href="style.css">
+	</head>
+
+<body>
+	<nav>
+		<a href="about.html">About Me</a> |
+		<a href="portfolio">Portfolio</a>
```

Comment:
Make sure your you include a file extension in your a tag for portfolio. So portfolio.html. It looks like it still works without, but that is the convention.